cloudevents/SDK.md

```diff
@@ -109,9 +109,16 @@ based on the parameters set, most importantly the CloudEvents spec version.
 
 #### Encode/Decode an Event
 
-Each SDK will support encoding and decoding an Event with regards to a transport
-and encoding. `Structured` encoding is the easiest to support, as it is just
-`json`, but `Binary` is fairly custom for each transport.
+Each SDK MUST support encoding and decoding an Event with regards to a transport
+and encoding:
+
+- Each SDK MUST support structured-mode messages for each transport that it
+  supports.
+- Each SDK SHOULD support binary-mode messages for each transport that it
```

Comment:
Would it break any SDK if we made this a MUST?

Reply:
Would be good to check. I'd personally be happy for it to be a must.

Reply:
@cloudevents/sdk-maintainers  ^^^ ?

Reply:
I just read the "Protobuf Event Format for CloudEvents," and as I see, there is no structured mode, right? - So then this will not work with protobuf.
https://github.com/cloudevents/spec/blob/main/cloudevents/formats/protobuf-format.md#3-data

Reply:
LGTM